csharp-design-patterns/UnitOfWork/UnitOfWork/DataAccess/Dapper/DapperContext.cs

```diff
@@ -0,0 +1,27 @@
+using System.Data.SqlClient;
+using Dapper;
+using UnitOfWork.Entities;
+
+namespace UnitOfWork.DataAccess.Dapper;
+
+public sealed class DapperContext : IDatabase
```

Comment:
You used `sealed` keyword here. Although I'm a big fan of the `sealed` approach, we should remove it from here since the article is not about it, to avoid confusing our readers and make our examples as simple as possible.

Reply:
Yeah, since I set up Rider to auto add it on file creation I have to remove it every time when I'm writing code for an article, and sometimes I miss it. Thanks for pointing out